src/layman/layer/rest_layer_style.py

```diff
@@ -0,0 +1,46 @@
+from flask import Blueprint, Response, current_app as app, g
+
+from layman import LaymanError
+from layman.util import check_uuid_decorator
+from layman.authn import authenticate
+from layman.authz import authorize_uuid_publication_decorator
+from layman.layer.geoserver import sld
+from layman.layer.qgis import wms as qgis_wms
+from layman import settings, util as layman_util
+from . import LAYER_REST_PATH_NAME, LAYER_TYPE
+
+bp = Blueprint('rest_layer_style', __name__)
+
+
+@bp.before_request
+@check_uuid_decorator
+@authenticate
+@authorize_uuid_publication_decorator(expected_publication_type=LAYER_TYPE)
+def before_request():
+    pass
+
+
+@bp.route(f"/{LAYER_REST_PATH_NAME}/<uuid>/style", methods=['GET'])
+def get(uuid):
+    app.logger.info(f"GET Layer Style, actor={g.user}")
+
+    info = layman_util.get_publication_info_by_uuid(uuid, context={'keys': ['style_type', 'access_rights']})
```

Comment:
```suggestion
    info = layman_util.get_publication_info_by_uuid(uuid, context={'keys': ['style_type']})
```

I think access_rights key is not needed here.